assets/chat/js/menus/ChatEmoteTooltip.js

```diff
@@ -23,11 +23,13 @@ export default class ChatEmoteTooltip extends ChatMenuFloating {
 
     this.emote.on('click', '.emote', (e) => {
       const value = this.chat.input.val().toString().trim();
-      this.chat.input
-        .val(
-          `${value + (value === '' ? '' : ' ') + e.currentTarget.innerText} `
-        )
-        .focus();
+      if (this.chat.emoteService.canUserUseEmote(this.chat.user, value)) {
+        this.chat.input
+          .val(
+            `${value + (value === '' ? '' : ' ') + e.currentTarget.innerText} `
+          )
+          .focus();
+      }
```

Comment:
I'm a little on the fence about this change. I suspect the user is just going to think the emote not printing in the input is a bug. I think I'd prefer if the emote just printed regardless of whether they meet the sub requirements.